LiskHQ/lisk-hub
# Improve url scheme - Closes #608

### What was the problem?
We are still using `main` route and old naming
### How did I fix it?
refactor code so it will use new terminology and no `main` route
### How to test it?
Check wallet and delegates urls if they match. Also make sure there is no `main` in any route.
Linked to https://github.com/LiskHQ/lisk-hub/issues/585
### Review checklist
- The PR solves #608
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
